tangos/live_calculation/builtin_functions/reassembly.py

```diff
@@ -27,6 +28,9 @@ def reassemble_initialisation(input, *options):
             options_values.append(option.proxy_value())
         else:
             raise TypeError("Options to 'reassemble' must be fixed numbers or strings")
-
-    input.set_extraction_pattern(
-        extraction_patterns.HaloPropertyValueWithReassemblyOptionsGetter(*options_values))
+    if isinstance(input,LiveProperty):
```

Comment:
Similar comment as above; you want a single `set_reassemble(*options_values)` which is implemented differently for `LiveProperty` and for `StoredProperty`

Reply:
Again, this seems to be pending resolution